grails-app/domain/org/pih/warehouse/inventory/OutboundStockMovementListItem.groovy

```diff
@@ -124,7 +125,16 @@ class OutboundStockMovementListItem implements Serializable {
     }
 
     Integer getLineItemCount() {
-        return requisition ? RequisitionItem.countByRequisition(requisition) : order ? OrderItem.countByOrder(order) : 0
+        if (requisition) {
+            RequisitionItem[] approvedItems = requisition.requisitionItems.findAll{ it ->
+                it.status == RequisitionItemStatus.APPROVED
```

Comment:
What happens when a requisition item has been substituted or modified?

Reply:
According to the [screen](https://i.imgur.com/6LKeqE5.png) if an item is substituted or its `quantityRevised` changes - this item still appears in `requsitionItems` with status `SUBSTITUTED`/`CHANGED` and another one is 'created' with `APPROVED` status (for example take a look at `10045` in terms of substituting the item and at `10026` in terms of changing the `quantityRevised` (there are 'two' 10045 and 'two' 10026) and it was double counted, so according to [this screen](https://i.imgur.com/Xzlab9k.png) it was showing wrong amount of items as '10' (additional two because of substitutions [18586 and 10045] and additional two because of changing the `quantityRevised` [10026 and 11587])